### PR TITLE
Removal of 3rd POV aiming reticles

### PR DESCRIPTION
**Pull request by Ollieollieolllie**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/sensor/fn_targetingFCRUpdate.sqf [hunk 1]
```diff
@@ -73,7 +73,7 @@ private _fcrTargets = [];
     if ((_type != FCR_TYPE_FLYER && _type != FCR_TYPE_HELICOPTER) && _fcrMode == 2) then {continue;};
     if ((vectorMagnitude velocityModelSpace _target) < 5 && _fcrMode == 2) then {continue;};
 
-    _fcrTargets pushBack [getPosAsl _target, _type, speed _target, _target];
+    _fcrTargets pushBack [aimpos _target, _type, speed _target, _target];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/sensor/fn_targetingFCRUpdate.sqf:76]: This doesn't seem related to the reticule?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Looks good! Though looks like you left a bit of FCR code in?

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/sensor/fn_targetingFCRUpdate.sqf (hunk 1 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/sensor/fn_targetingFCRUpdate.sqf]: fixed

**Review by mattysmith22 — APPROVED** [review]
